### PR TITLE
[ADD] Travis and branding

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Thanks! :+1: 

I'll merge when Travis finishes (now is very busy with the mass-commits done)

**Comment by pedrobaeza** [comment]
Please add `sudo: False` for enabling Travis container mode for being faster.

**Comment by pedrobaeza** [comment]
Ouch, Travis complains about the inexistance of scripts folder. Please check.

**Comment by StefanRijnhart** [comment]
Thanks Travis, for reminding me to copy over the scripts!

**Comment by StefanRijnhart** [comment]
Green now.

File: scripts/migrate.py [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+             '| pg_restore --no-password --dbname=%(db_new)s') %
+            {'db_old': db_old, 'db_new': db_new}
+        )
+
+        if password_set:
+            del os.environ['PGPASSWORD']
+    return db_new
+
+migrations = {
+    '8.0': {
```
Comment (pedrobaeza) [inline scripts/migrate.py:79]: No 9.0 entry?

**Comment by StefanRijnhart** [comment]
Sorry, I never use that script (but I should ;-). Has now.

**Comment by pedrobaeza** [comment]
:+1: 

@hbrunn, do you have any comments, or may I merge?

**Pull request merged**